### ISSUE TITLE
Please add complete example into ReadMe

### ISSUE THREAD

**Issue by harrison314** [cdiggins/parakeet#2]
Hello,
I tried to start using parakeet because its API looks good and is practically used.

Well, at the very beginning I ran into the problem of which nuggets to install in order for the example given on CodeProject to work. I solved that.

I had a problem with practical use, I corrected the grammar, but the _ReadMe_ lacks how to parse the input, how to pass AST or tokens,...

Could you add a complete example to the _ReadMe_, for example a simple calculator? It would be enough to show the grammar, show its notation in _Parakeet_, how to parse the input, display the AST/tokens on the console, calculate the result of the math formula (eg. `32 + 14 * (2 + 3) - 1`).

I mean the grammar for the calculator, which is often used in examples for parsers:
```
E ::= E '+' T | E '-' T | T
T ::= T '*' F | T '/' F | F
F ::= number | '(' E ')'
number ::= `[1-9]\d*`
```

**Comment by Palatis** [comment]
you might want to take a look at [Parakeet.Grammars](https://github.com/ara3d/parakeet/Parakeet.Grammars) for some examples.
there are easy examples like the [CsvGrammar](https://github.com/ara3d/parakeet/blob/master/Parakeet.Grammars/CsvGrammar.cs) and [SExpressGrammar](https://github.com/ara3d/parakeet/blob/master/Parakeet.Grammars/SExpressionGrammar.cs), or some more advanced examples such as [CSharpGrammar](https://github.com/ara3d/parakeet/blob/master/Parakeet.Grammars/CSharpGrammar.cs) or [JsonGrammar](https://github.com/ara3d/parakeet/blob/master/Parakeet.Grammars/JsonGrammar.cs).

basically you want to extends `BaseCommonGrammar`, and define your language by defining rules.
